doc/content/whats-new/3.33.1.md

```diff
@@ -0,0 +1,18 @@
+---
+date: 2025-02-10T15:41:53Z
+title: "3.33.1"
+featured: { added: ["Add live data split view tutorial to the Console."] }
```

Comment:
Please add all three items in the `added` list here.

Reply:
done